views/spaces/new.erb

```diff
@@ -1,4 +1,5 @@
 <form action='/spaces' method='post'>
   <input type='text' name='space_name' placeholder='Name your space'>
+  <input type='text' name='description' placeholder='Description'>
   <input type='submit' value='Submit'>
```

Comment:
We have used labels and placeholders for our forms - we should rackup and take a look at each form and decide as a group which one we want to go with and ensure consistency across forms